### PR TITLE
Controllers testing

### PR DESCRIPTION
**Pull request by Sboursen**
# Description

Milestone #2

- [x] Set up Rspec
- [x] Implement tests for Users controller
- [x] Implement tests for Posts controller

Cheers,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ## Milestones
```
Comment (SajjadAhmad14) [inline README.md:6]: Please add `rspec` command in README so a user can run the tests.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Soufiane, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (Sboursen) [reply README.md:6]: Hi @SajjadAhmad14

Per the [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md), there is nothing about adding `rspec` to the `readme`.

image

I agree that It is a nice addition, but you could just mentioned it as a [RECOMMANDED], no need for a total rejection!

Best,

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Sboursen, 

In my opinion, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by nicupop729 — COMMENTED** [review]
Hello @Sboursen 👋,

_This is an additional review as a result of the second opinion request._

It is not the usual answer to slow, just that we noticed that the second reviewer approved the pull request so we didn't consider it urgent anymore.

I am glad that everything worked out fine in the end!

Keep rocking 🚀!

**Pull request merged**